source/_integrations/dynalite.markdown

```diff
@@ -0,0 +1,140 @@
+---
+title: "Philips Dynalite"
+description: "Instructions on setting up Philips Dynalite within Home Assistant."
+logo: dynalite.png
+ha_category:
+  - Hub
+  - Light
+ha_iot_class: Local Push
+ha_qa_scale: No Score
+featured: true
+ha_release: "0.60"
```

Comment:
```suggestion
ha_release: 0.101
```

Reply:
thanks. will make these changes now. i used hue as a base as there are many similarities, but didn't notice. will change and push now

Reply:
done. uploaded to fork

Reply:
done